dataset/data.go

```diff
@@ -197,6 +199,11 @@ type Metadata struct {
 	DatasetLinks Links `json:"dataset_links,omitempty"`
 }
 
+type TopicTagObject struct {
```

Comment:
Maybe we could find a better name for this type. Is there any benefit in having `Object` in it? How about simplifying it to `Topic`?

Reply:
`Topic` it is then :white_check_mark: